benchmarks/octane/octane.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const fs = require('fs');
+const vm = require('vm');
+
+global.load = load;
+global.read = read;
+global.print = console.log;
+
+process.chdir('octane')
```

Comment:
@bnoordhuis The intention is to chdir into the octane directory that had been git cloned - so run.js is in the right place to load all the other tests.  This was based on already being in the benchmarking/benchmarks/octane directory and then cloning octane from there - which is possibly not the best/most desirable course of action, depending on what the normal course of action is in the jenkins scripts